### PR TITLE
Caching pattern: Cache-Aside

### PR DESCRIPTION
**Pull request by christofferh**
Implementation of https://github.com/iluwatar/java-design-patterns/issues/440, (+ minor style fixes)

I'll update documentation & UML once the solution has been reviewed and approved.

### CODE REVIEW

**Comment by iluwatar** [comment]
@christofferh the implementation looks good. Please update the documentation so we can proceed with the merge.

File: caching/src/main/java/com/iluwatar/caching/App.java [hunk 1]
```diff
@@ -136,4 +137,26 @@ public void useReadThroughAndWriteBehindStrategy() {
     AppManager.find("004");
     System.out.println(AppManager.printCacheContent());
   }
+
+  /**
+   * Cache-Aside
+   */
+  public void useCacheAsideStategy() {
+    System.out.println("# CachingPolicy.ASIDE");
```
Comment (chinnachalam) [inline caching/src/main/java/com/iluwatar/caching/App.java]: Is there any reason for not using logging framework like SLF4J or LOG4J Or Common logging?

**Comment by christofferh** [comment]
@iluwatar, docs and and diagram added. 

Sidenote: +1 for https://github.com/iluwatar/java-design-patterns/issues/190, as an intellij user the diagram generation part was not that enjoyable :)

**Comment by iluwatar** [comment]
Thank you @christofferh for this addition :+1:

**Pull request merged**